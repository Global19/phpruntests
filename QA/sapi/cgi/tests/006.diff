006+ <font size='1'><table dir='ltr' border='1' cellspacing='0' cellpadding='1'>
007+ <tr><th align='left' bgcolor='#f57900' colspan="5"><span style='background-color: #cc0000; color: #fce94f; font-size: x-large;'>( ! )</span> Parse error: syntax error, unexpected T_PRIVATE, expecting '{' in /mnt/workspace/php53_tmp/sapi/cgi/tests/006.test.php on line <i>5</i></th></tr>
006- <b>Parse error</b>: %s expecting %s{%s in <b>%s006.test.php</b> on line <b>5</b><br />