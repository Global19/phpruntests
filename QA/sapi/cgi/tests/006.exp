string(%d) "No syntax errors detected in %s006.test.php
"
string(%d) "No input file specified.
"
string(%d) "<br />
<b>Parse error</b>: %s expecting %s{%s in <b>%s006.test.php</b> on line <b>5</b><br />
Errors parsing %s006.test.php
"
Done