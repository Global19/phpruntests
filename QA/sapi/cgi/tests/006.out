string(82) "No syntax errors detected in /mnt/workspace/php53_tmp/sapi/cgi/tests/006.test.php
"
string(25) "No input file specified.
"
string(438) "<br />
<font size='1'><table dir='ltr' border='1' cellspacing='0' cellpadding='1'>
<tr><th align='left' bgcolor='#f57900' colspan="5"><span style='background-color: #cc0000; color: #fce94f; font-size: x-large;'>( ! )</span> Parse error: syntax error, unexpected T_PRIVATE, expecting '{' in /mnt/workspace/php53_tmp/sapi/cgi/tests/006.test.php on line <i>5</i></th></tr>
Errors parsing /mnt/workspace/php53_tmp/sapi/cgi/tests/006.test.php
"
Done